src/screens/Register/Register.jsx

```diff
@@ -28,6 +29,13 @@ const Register = ({ image }) => {
   const selfHosted = useSelector((state) => state.login.selfHosted);
   const languages = useSelector((state) => state.language.languages);
 
+  const privacyText = t("screens.register.readPrivacy");
+  const [privacyTextBefore, privacyTextLink, ...privacyTextAfter] =
```

Comment:
does that work, or is that throwing an error, if there is nothing after? And what happens if you add some braces afterwards? E.g. `I have read the [Privacy Policy] abc [test].`